docs/content.zh/release-notes/flink-1.18.md

```diff
@@ -0,0 +1,152 @@
+---
+title: "Release Notes - Flink 1.18"
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Release notes - Flink 1.18
+
+These release notes discuss important aspects, such as configuration, behavior or dependencies,
+that changed between Flink 1.17 and Flink 1.18. Please read these notes carefully if you are
+planning to upgrade your Flink version to 1.18.
+
+
+### Build System
+
+#### Support Java 17 (LTS)
+
+##### [FLINK-15736](https://issues.apache.org/jira/browse/FLINK-15736)
+Apache Flink was made ready to compile and run with Java 17 (LTS). This feature is still in beta mode. 
+Issues should be reported in Flink's bug tracker.
+
+
+### Table API & SQL
+
+#### Unified the max display column width for SqlClient and Table APi in both Streaming and Batch execMode
+
+##### [FLINK-30025](https://issues.apache.org/jira/browse/FLINK-30025)
+Introduction of the new ConfigOption DISPLAY_MAX_COLUMN_WIDTH (table.display.max-column-width) 
+in the TableConfigOptions class is now in place. 
+This option is utilized when displaying table results through the Table API and sqlClient. 
```

Comment:
```suggestion
This option is utilized when displaying table results through the Table API and SQL Client. 
```
I noticed in docs that most of the time it is either `SQL Client` or `sql client`, probably would make sense to follow same principle